packages/webpack-config-single-spa-ts/lib/webpack-config-single-spa-ts.js

```diff
@@ -1,25 +1,23 @@
 const webpackConfigSingleSpa = require("webpack-config-single-spa");
 const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
-const merge = require("webpack-merge");
+const { mergeWithCustomize } = require("webpack-merge");
 
 const modifyConfig = (opts, webpackConfig) => {
-  const typescriptPath =
-    opts.typescriptPath ||
-    require.resolve("typescript", { paths: [process.cwd()] });
+  const merge = mergeWithCustomize({
+    customizeArray(first, second, key) {
+      return first.concat(second);
+    },
+  });
 
-  return merge.strategy({
-    entry: "replace",
-    plugins: "append",
-    "resolve.extensions": "append",
-  })(webpackConfig, {
+  return merge(webpackConfig, {
     entry: webpackConfig.entry.replace(
       ".js",
       opts.framework === "react" ? ".tsx" : ".ts"
     ),
     plugins: [
       opts.webpackConfigEnv && opts.webpackConfigEnv.analyze
         ? false
-        : new ForkTsCheckerWebpackPlugin({ typescript: typescriptPath }),
```

Comment:
This webpack plugin stopped supporting custom typescript paths. As a result, I added typescript as a hard dependency of webpack-config-single-spa-ts, and it seems to be working.